Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/Redirect.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
```

Comment:
Licence header missing

Reply:
It's missing for all files in the package...